client/my-sites/current-site/index.jsx

```diff
@@ -152,7 +93,8 @@ class CurrentSite extends Component {
 					</div>
 					: <AllSites />
 				}
-				{ ! isJetpack && this.getDomainWarnings() }
+				<AsyncLoad require="my-sites/current-site/domain-warnings"
+					placeholder={ <EmptyComponent /> } />
```

Comment:
I'm happy to introduce new prop in `AsyncLoad`, that might be useful whenever you load code that displays conditionally.